### DISCUSSION TITLE
Exploit to crash any extension

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#890] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Navigate to the manifest page of the extension you want to crash (chrome://extension[extension id]/manifest.json)
Then, be sure to copy chrome://hang in your clipboard. Type in chrome://kill, reload as fast as possible, and then paste in chrome://hang. This needs to be done quickly or it will not work. I don’t know if I’ve posted this here before, but last I checked it works

EDIT: I noticed that this exploit was recently listed (also named LTMEAT, although it wasn’t discovered by the person who claims)

**Comment by leahanderson1** [comment]
TLDR: it was

**Comment by AshtonDavies** [comment]
Some believe it wasn't discovered by Bypassi#7037, but I have yet to see any proof and I think he deserves the credit better than any other human being.

**Reply by ghost** [reply]
It wasn’t, I had been implementing it before it was widely heard of

**Reply by AshtonDavies** [reply]
Ah, I see you're here from kajigs. Yes, I've seen yours, and I respect that it was in development first, but I'd still say that Bypassi has done more of the work in terms of explanation and that kind of stuff before your method was really established/known.

Now, if he literally straight up stole it from you intentionally without credit then I am mistaken here.

**Comment by DankFireR** [comment]
this has already been patched im p sure
